cli/running/running.go

```diff
@@ -348,7 +348,7 @@ func loadInstanceConfig(configPath, instName string,
 
 // collectInstancesFromAppDir collects instances information from application directory.
 func collectInstancesFromAppDir(appDir string, selectedInstName string,
-	integrityCtx integrity.IntegrityCtx, instancesScriptsRequired bool) (
+	integrityCtx integrity.IntegrityCtx, configRequired bool) (
```

Comment:
The behavior is too complex, not obvious and fault-tolerate now. Probably we need to skip a config load at all if a data from config is not required.

```suggestion
	integrityCtx integrity.IntegrityCtx, skipClusterConfig bool) (
```

Please, try to skip a config load in a **separate** commit.